client/packages/common/src/intl/locales/en/common.json

```diff
@@ -117,6 +117,7 @@
   "label.enrolment-program": "Program",
   "label.enrolment-patient-id": "Enrolment Patient Id",
   "label.enrolment-datetime": "Enrolment Date",
+  "label.encounters": "Encounters",
```

Comment:
with the shift to the shared tabs component - is now looking in 'common' for these